R/getTsMeasure.R

```diff
@@ -0,0 +1,86 @@
+#' @title Get Time Series Measures
+#' @description Calculate various time series measures (e.g., ACF, PACF) 
+#' from a specified column in the `colData` of a `TreeSE` object.
+#' 
+#' @param x A
+#' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object.
+#' @param time.field \code{Character scalar}. Specifies the name of the
+#' time series field in `colData`. (Default: \code{"Time.hr"})
+#' @param measures \code{Character vector}. Specifies the time series measures 
+#' to compute. Supported measures include "acf", "pacf","Box.test", and "arima".
+#' (Default: \code{ ""acf""})
```

Comment:
Documentation is lacking info. For instance, arima model has lots of parameters that user wants to modify. However, now it is very hard to know what are those parameter names